public/js/dashboards/dashboards-view.controller.js

```diff
@@ -120,8 +137,15 @@
             const modal = $uibModal.open({
                 component: 'appPdfExportSettingsModal',
             });
-
+            const filters = {};
+            if (vm.prompts) {
+                for (const key of Object.keys(vm.prompts)) {
+                    filters[key] = vm.prompts[key].criterion;
+                }
+            }
+            $location.search('filters', JSON.stringify(filters));
```

Comment:
I think it would be more useful to set URL parameters at "repaint" time. This way, everytime the reports are recalculated, the URL is in sync with what is displayed, and users can reload the page and get the same result.
If it is done only at export time, users who want to share the URL will have to export the report first, even if they don't want the export.

Reply:
I change it too, thanks